sessions/recap-project-5/example-implementation/pages/art-pieces/index.js

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+import ArtPiecesPreview from "../../components/ArtPiecesPreview";
+
+const ImageGrid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(1fr);
+  grid-gap: 5px;
+  width: 600px;
+  height: 100%;
+`;
+
+export default function ArtPieces({
```

Comment:
According to User Story 1, there should be a component called `ArtPieces`: if we want to do this, I have no idea how to handle the naming issues. What would be the name of this page function, then? 🤷

Reply:
@aclaes

Reply:
Yup, we should create a component called `ArtPieces` with everything in it.

We should not advise to define a bunch of styled components inside of a Next.js page

Regarding the naming conflict: 

```suggestion
export default function ArtPiecesPage({
```

(We could add the suffix `Page` to all pages ;-)